frameworks/kafka/universe/config.json

```diff
@@ -490,14 +490,14 @@
           "inter_broker_protocol_version":{
             "type":"string",
             "title":"inter.broker.protocol.version",
-            "description":"Specify which version of the inter-broker protocol will be used, which must align with log.message.format.version. This is typically bumped after all brokers were upgraded to a new version. Example of some valid values are: 0.8.0, 0.8.1, 0.8.1.1, 0.8.2, 0.8.2.0, 0.8.2.1, 0.9.0.0, 0.9.0.1, 0.10.0.0. Check ApiVersion for the full list.",
-            "default":"0.10.0.0"
+            "description":"Specify which version of the inter-broker protocol will be used, which must align with log.message.format.version. This is typically bumped serially one broker at a time, *after*all brokers were upgraded to a new version. Example of some valid values are: 0.8.0, 0.8.1, 0.8.1.1, 0.8.2, 0.8.2.0, 0.8.2.1, 0.9.0.0, 0.9.0.1, 0.10.0.0, 0.10.1.x, 0.10.2.x, 0.11.0.0. Check ApiVersion for the full list.",
```

Comment:
There is a space missing in `*after*all`. Also, most other json we have (e.g. resources.json) has a space following a ":" whereas this file does not. Is there a specific reason for the difference?